crates/turborepo-auth/src/error.rs

```diff
@@ -19,4 +26,6 @@ pub enum Error {
     FailedToValidateSSOToken(#[source] turborepo_api_client::Error),
     #[error("failed to make sso token name")]
     FailedToMakeSSOTokenName(#[source] io::Error),
+    #[error("config directory not found")]
```

Comment:
In a future PR I want to move this into the `turborepo_dirs` crate, but for now it can live here.